src/components/content/ContentMeetup.tsx

```diff
@@ -1,85 +1,92 @@
-        width: 100%,
-    `;
-    const PostContainer = styled.div`
-        align-items: center;
-        display: flex;
-        flex-direction: row;
-        flex-wrap: wrap;
-        justify-content: center;
-        width: 100%;
-        cursor: pointer;
-    `;
+export default function ContentMeetup(props: IContentPostProps) {
+    const classes = useStyles();
 
     const sendToMeetupId = (event: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
-        props.onClick(props.content._id);
+        props.onClick(props.content._id as string);
         event.preventDefault();
     };
 
-    const coverImage = props.content.coverImage && <ImagePost>
-        <img src={props.content.coverImage} alt="presenting" style={{ width: '100%' }} />
-    </ImagePost>;
-
-    const userAvatarSrc = ''; // 'https://ipfs.io/ipfs/' + props.content.author.image[0].contentUrl['/'];
-
     return (
-        <PostContainer onClick={sendToMeetupId}>
-            <Panel shaded={true} bordered={true} bodyFill={true} style={mainPanelStyle}>
-                {coverImage}
-                <Panel header={props.content.title} >
-                    <div className="show-grid">
-
-                        <FlexboxGrid align="middle">
-                            <FlexboxGrid.Item colspan={3}>
-                                <div style={{ lineHeight: 0 }}>
-                                    <Avatar circle={true} src={userAvatarSrc} />
-                                </div>
-                            </FlexboxGrid.Item>
-                            <FlexboxGrid.Item colspan={18}>
-                                <div style={{ lineHeight: 1.5 }}>
-                                    <p><b>{props.content.author}</b></p>
-                                    <p>
-                                        <Icon icon="calendar-check-o" />
-                                        &nbsp;{format('dd/MM/yyyy hh:mm', new Date(props.content.date * 1000))}
-                                    </p>
-                                </div>
-                            </FlexboxGrid.Item>
-                        </FlexboxGrid>
-                    </div>
-                    {props.content.description.split('\n').map((i, index) => <p key={index}>{i}</p>)}
-                    <Button>Continuar a ler</Button>
-                </Panel>
-            </Panel>
-        </PostContainer>
+        <div style={{ margin: '50px' }} onClick={sendToMeetupId}>
+            <Card className={classes.root}>
+                <CardHeader
+                    avatar={
+                        <Avatar
+                            aria-label="recipe"
+                            className={classes.avatar}
+                        >
+                            {props.content.author.substr(0, 2)}
+                        </Avatar>
+                    }
+                    title={props.content.title}
+                    subheader={format('MM dd, yyyy - hh:mm', new Date(props.content.date * 1000))}
```

Comment:
Desculpa, as datas erradas são culpa minha. Aqui fica a coreção. Faz a mesma alteração ara os posts. Obrigado

```suggestion
                    subheader={format('MM dd, yyyy - hh:mm', new Date(props.content.date))}
```